### ISSUE TITLE
MD_PLANT's damage calculation and its interaction with SC_PNEUMA and SC_SAFETYWALL

### ISSUE THREAD

**Issue by AnisotropicDefixation** [HerculesWS/Hercules#1109]
Currently monsters with this mode do not benefit from those SC statuses at all, always receiving 1 damage, this applies to Emperium and battleground stones as well, which is a rather big deal.

**Comment by Jedzkie** [comment]
Is this the official behavior?

**Comment by AnisotropicDefixation** [comment]
As far as I know nothing regarding this was changed with renewal and SWing emperium has been an integral part of the game since the dawn of times.

**Comment by Playtester** [comment]
rA also has some weird code making Safety Wall not work in GVG grounds and Emperium. I couldn't find out any source why this was done...

**Comment by AnisotropicDefixation** [comment]
Yeah, I actually adapted your rA commit that fixes offhand and double/triple attack interaction with plants and thought it could be at fault due to the fixed  = 1 damage values but nope, same issue with clean source.
This is rather important and should be looked into.

**Comment by Playtester** [comment]
My plant code itself should be fine, it's just that I didn't touch the exception for Emperium because I thought there must be some reason why someone put them in there in the first place.

Unfortunately I'm not really a PVP player so I have no means to test on active WoE/BG grounds.

If you just spawn an Emperium on an Aegis server Safety Wall works just fine, though...
